feat(sig): Submit proposal for a CF SIG (#137)
spinnaker/governance: sig-cf/README.md

# SIG: CF

This SIG focuses on the current Cloud Foundry (CF) clouddriver provider for Spinnaker.

## Meetings

* Regular SIG Meeting

>   * Last Thursday of each month, at 1 PM EST / 10AM PST

Reviewer: ```suggestion
  * Last Thursday of each month, at 10 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
```

  
  [Agenda](https://docs.google.com/document/d/1TouFx5ykNCcZxNSD2JhUiRliZDXwxZ6sx239rigYdxw/edit?usp=sharing)

## Leadership

* Zach Smith - Armory (https://github.com/zachsmith1)
* Ryan Pei - Armory (https://github.com/ryanpei)

## Contact

> * [Slack](https://spinnakerteam.slack.com/archives/C4ENF2E1W)

Reviewer: The convention for the other SIGs is to have the channel be called `#sig-{name}`; would it make sense to use that here to for discoverability?

Author: good idea!

## Scope

*Status and Alignment*

>   * [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry)

Reviewer 2: This appears to be narrowly scoped to just cloudfoundry in `clouddriver` ... I assume this SIG will also cover CF-related issues that may pop up in the UI and orchestration tiers?

Author: Yes, is this more clear?
```suggestion
  * Issues in [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry) and other Spinnaker services which affect deployments to CloudFoundry
```

>     * Review issues and PRs

Author: remove this line if the suggestion above is preferred
```suggestion
```
